front/components/AppLayout.tsx

```diff
@@ -61,213 +61,217 @@ export default function AppLayout({
                         </div>
-                      </li>
-                    ) : null}
-                  </ol>
-                </nav>
-                <div className="static inset-auto hidden flex-initial items-center pr-4 md:flex">
-                  <Link href="https://docs.dust.tt">
-                    <Button>
-                      <>
-                        <ArrowRightCircleIcon className="-ml-1 mr-2 h-4 w-4" />
-                        View Documentation
-                      </>
-                    </Button>
-                  </Link>
-                </div>
-                {user ? (
-                  <div className="static inset-auto right-0 flex flex-initial items-center pr-2">
-                    <Menu as="div" className="relative">
-                      <div>
-                        <Menu.Button className="focus:outline-nonek flex rounded-full bg-gray-800 text-sm">
-                          <span className="sr-only">Open user menu</span>
-                          <img
-                            className="h-8 w-8 rounded-full"
-                            src={
-                              user.image
-                                ? user.image
-                                : "https://gravatar.com/avatar/anonymous"
-                            }
-                            alt=""
-                          />
-                        </Menu.Button>
                       </div>
-                      <Menu.Items className="absolute right-0 z-10 mt-2 origin-top-right rounded-md bg-white py-1 shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none">
-                        <Menu.Item>
-                          {({ active }) => (
-                            <a
-                              href="https://docs.dust.tt"
-                              className={classNames(
-                                active ? "bg-gray-50" : "",
-                                "block px-4 py-2 text-sm text-gray-700"
+                    </div>
+                    <nav className="ml-1 flex h-12 flex-1">
+                      <ol role="list" className="flex items-center space-x-1">
+                        <li>
+                          <div className="flex items-center">
+                            <ChevronRightIcon
+                              className="mr-1 h-5 w-5 shrink pt-0.5 text-gray-400"
+                              aria-hidden="true"
+                            />
+                            {user && user.workspaces.length > 1 ? (
+                              <WorkspacePicker
+                                user={user}
+                                workspace={owner}
+                                readOnly={false}
+                                onWorkspaceUpdate={(workspace) => {
+                                  if (workspace.id === owner.id) {
+                                    if (dataSource) {
+                                      router.push(`/w/${workspace.sId}/ds`);
```

Comment:
might be worth to pre-emptively `void` those wdyt ?

Reply:
yes wilco!